app/src/main/kotlin/com/wire/android/ui/home/conversations/MessageExpiration.kt

```diff
@@ -193,6 +193,18 @@ class SelfDeletionTimer(private val context: Context) {
                 if (timeLeft.inWholeSeconds != 0L) timeLeft -= interval
             }
 
+            /**
+            * if the time elapsed ratio is between 0.50 and 0.75
+            * we want to change the value proportionally to how much
+            * time is left between 0.50 and 0.75, we doing that by dividing
+            * how much time is elapsed since half of the total expire after time by
+            * the "time slice" that fits between 0.5 and 0.75
+            * for example. [expireAfter] = 10 sec, timeElapsed = 6 sec
+            * quarterTimeLeftSlice = 2.5 sec, halfTimeSlice = 5 sec
+            * durationInBetweenHalfTimeAndQuarterSlice = 5 sec - 2.5 sec = 2.5 sec
+            * timeElapsedFromHalfTimeSlice = 6 sec - 5 sec = 1 sec
+            * alpha value is equal to the ratio = 1 / 2.5 = 0.4
```

Comment:
Instead of copying `0.5` and `0.75`, please use something like:

```kotlin
            * if the time elapsed ratio is between [LOW_TIME_ELAPSED_RATIO_BOUNDARY_FOR_PROPONTIAL_ALPHA_CHANGE] and [HALF_RATIO_VALUE]
            * we want to change the value proportionally to how much
            * time is left between [LOW_TIME_ELAPSED_RATIO_BOUNDARY_FOR_PROPONTIAL_ALPHA_CHANGE] and [HALF_RATIO_VALUE], we doing that by dividing

```

So if the values change, there's no need to update the docs. Also, it helps to understand what these values are for.

Keep in mind that I just copied/pasted some names in there.